python/ray/tests/test_placement_group.py

```diff
@@ -1854,5 +1856,75 @@ def get_gpu(self):
     assert len(gpu_ids_res) == 4
 
 
+@pytest.mark.parametrize("execution_number", range(3))
+def test_placement_group_remove_stress(ray_start_cluster, execution_number):
```

Comment:
@sasha-s this is actually a great case where forced interleaving could be helpful for reproducing race conditions. GCS could also be a place to see if this kind of technique would be helpful.

Reply:
thank you for the pointer @ericl